### ISSUE TITLE
Electron mode, ReferenceError: onBuildChange is not defined

### ISSUE THREAD

**Issue by baryon** [quasarframework/quasar#8771]
**Describe the bug**

when dev electron mode app, an error was raised.

```
(node:22143) UnhandledPromiseRejectionWarning:   ReferenceError: onBuildChange is not defined
  
  - quasar-dev:261 goLive
    [bsvwallet]/[@quasar]/app/bin/quasar-dev:261:26
  
  - task_queues.js:97 processTicksAndRejections
    internal/process/task_queues.js:97:5
```

we need fix quasar-dev:261: like
```
  if (['cordova', 'capacitor', 'electron', 'bex', 'pwa'].includes(argv.mode)) {
    const ModeRunner = require('../lib/' + argv.mode)
    ModeRunner.init(ctx, quasarConfFile.onBuildChange)
    runMode = () => ModeRunner.run(quasarConfFile, argv)
  }

```

it will work.

**To Reproduce**

% quasar create bsvwallet --branch next

Create quasar 2 app

```
? Project name (internal usage for dev) xxx
? Project product name (must start with letter if building mobile apps) Quasa
r App
? Project description A Quasar Framework app
? Author user <user@>
? Pick your CSS preprocessor: SCSS
? Check the features needed for your project: ESLint (recommended), TypeScrip
t, Vuex, Axios, Vue-i18n
? Pick a component style: Composition
? Pick an ESLint preset: Prettier
? Continue to install project dependencies after the project has been created
? (recommended) NPM
```
then add electron mode

```
quasar mode add electron
```

then dev
```
quasar dev -m electron
```

**Expected behavior**

No error

**Screenshots**

**Platform (please complete the following information):**

```
 Dev mode.......... electron
 Pkg quasar........ v2.0.0-beta.12
 Pkg @quasar/app... v3.0.0-beta.11
 Debugging......... enabled

 Configured browser support (at least 88.01% of global marketshare):
 · Chrome for Android >= 89
 · Firefox for Android >= 86
 · Android >= 89
 · Chrome >= 79
 · Edge >= 86
 · Firefox >= 78
 · iOS >= 10.3
 · Opera >= 69
 · Safari >= 11
```

**Comment by samk-dev** [comment]
I am having the same issue

**Comment by rstoenescu** [comment]
Fix is available in "@quasar/app" v3.0.0-beta.12
